src/client/cli/formatter/json_formatter.cpp

```diff
@@ -15,9 +15,9 @@
  *
  */
 
-#include <multipass/cli/json_formatter.h>
-
+#include <multipass/cli/client_common.h>
 #include <multipass/cli/format_utils.h>
+#include <multipass/cli/json_formatter.h>
```

Comment:
Please keep "self" header at top.

Reply:
These still need reordering back.

Reply:
I run clang-format on this and it always complains...